### ISSUE TITLE
Uncaught TypeError: T.getEditorLinter is not a function

### ISSUE THREAD

**Issue by denizkenan** [mulesoft/api-workbench#447]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.15.0 x64
**Electron**: 1.3.13
**OS**: Mac OS X 10.11.6
**Thrown From**: [api-workbench](https://github.com/mulesoft/api-workbench) package 0.8.44

### Stack Trace

Uncaught TypeError: T.getEditorLinter is not a function

```
At /Users/kenandeniz/.atom/packages/api-workbench/main.js:2

TypeError: T.getEditorLinter is not a function
  at Object.e.relint (/packages/api-workbench/main.js:2:7023)
  at /packages/api-workbench/main.js:1:25107)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:259:29)
  at CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:3:59)
  at CommandRegistry.module.exports.CommandRegistry.dispatch (/app.asar/src/command-registry.js:160:19)
  at AtomEnvironment.module.exports.AtomEnvironment.dispatchContextMenuCommand (/app.asar/src/atom-environment.js:1168:28)
  at EventEmitter.outerCallback (/app.asar/src/application-delegate.js:325:25)
  at emitThree (events.js:116:13)
  at EventEmitter.emit (events.js:194:7)

```

### Commands

```
     -0:46.2.0 api-workbench:revalidate (div.line.cursor-line)
     -0:39.3.0 core:copy (atom-notification.fatal.icon.icon-bug.native-key-bindings.has-detail.has-close.has-stack)
```

### Non-Core Packages

```
api-workbench 0.8.44 
atom-material-ui 1.3.9 
busy-signal 1.3.0 
haskell-grammar 0.4.0 
highlight-selected 0.13.1 
intentions 1.1.2 
language-babel 2.57.1 
language-graphql 0.8.0 
language-ini 1.19.0 
language-lua 0.9.11 
language-ocaml 1.1.2 
language-swift 0.5.0 
language-thrift 1.0.2 
linter 2.1.2 
linter-ui-default 1.2.1 
merge-conflicts 1.4.4 
nuclide 0.211.0 
nuclide-format-js 0.0.36 
set-syntax 0.3.2 
sort-lines 0.14.0 
tool-bar 1.1.0 
```

**Comment by ddenisenko** [comment]
Could you please post a reproduction sequence? What were you doing when this error happened?

**Comment by kkhodykin** [comment]
I also have this issue, it happens when click "Relaunch validation"

**Comment by ddenisenko** [comment]
@kkhodykin Thanks.

Tried to reproduce with calling this action, no luck.

 Could you please provide as many details regarding of how to reproduce it as you can? All clicks are important.

**Comment by altvod** [comment]
Same here: just open Atom > right-click > click Relaunch validation > error appears
Same traceback

**Comment by ddenisenko** [comment]
Okay, finally reproduced, it looks like this is happening on the latest version of linter due to some linter changes.

Thank you guys.

**Comment by dreamflyer** [comment]
will be published in v0.8.45.
